packages/golden_toolkit/lib/src/golden_builder.dart

```diff
@@ -99,6 +99,23 @@ class GoldenBuilder {
     ));
   }
 
+  ///  [addScenarioContextWithContext] will add a test with BuildContext GoldenBuilder
+  /// use as:
+  /// ..addScenarioContextWithContext(
+  ///   'Test with context',
+  ///   (context) {
+  ///     return Container();
+  ///   },
+  /// )
+  void addScenarioContextWithContext(
+      String name, Widget Function(BuildContext context) fn)  {
+    scenarios.add(_Scenario(
```

Comment:
can this delegate to addScenario() instead of reimplementing?

Reply:
yes